resources/locations.json

```diff
@@ -0,0 +1,212 @@
+[
+  {
+    "name": "MENSA_ARCISSTR",
+    "long_name": "Mensa Arcisstra\u00dfe",
+    "site": {
+      "address": "Arcisstra\u00dfe 17, M\u00fcnchen",
+      "latitude": 48.14742,
+      "longitude": 11.56722
+    },
+    "url_id": 421
+  },
+  {
+    "name": "MENSA_GARCHING",
+    "long_name": "Mensa Garching",
+    "site": {
+      "address": "Boltzmannstra\u00dfe 19, Garching",
+      "latitude": 48.268132,
+      "longitude": 11.672263
+    },
+    "url_id": 422
+  },
+  {
+    "name": "MENSA_LEOPOLDSTR",
+    "long_name": "Mensa Leopoldstra\u00dfe",
+    "site": {
+      "address": "Leopoldstra\u00dfe 13a, M\u00fcnchen",
+      "latitude": 48.156311,
+      "longitude": 11.582446
+    },
+    "url_id": 411
+  },
+  {
+    "name": "MENSA_LOTHSTR",
+    "long_name": "Mensa Lothstra\u00dfe",
+    "site": {
+      "address": "Lothstra\u00dfe 13d, M\u00fcnchen",
+      "latitude": 48.153989,
+      "longitude": 11.552424
+    },
+    "url_id": 431
+  },
+  {
+    "name": "MENSA_MARTINSRIED",
+    "long_name": "Mensa Martinsried",
+    "site": {
+      "address": "Gro\u00dfhaderner Stra\u00dfe 44, Plategg",
+      "latitude": 48.109824,
+      "longitude": 11.460006
+    },
+    "url_id": 412
+  },
+  {
+    "name": "MENSA_PASING",
+    "long_name": "Mensa Pasing",
+    "site": {
+      "address": "Am Stadtpark 20, M\u00fcnchen",
+      "latitude": 48.141568,
+      "longitude": 11.451119
+    },
+    "url_id": 432
+  },
+  {
+    "name": "MENSA_WEIHENSTEPHAN",
+    "long_name": "Mensa Weihenstephan",
+    "site": {
+      "address": "Maximus-von-Imhof-Forum 5, Freising",
+      "latitude": 48.39959,
+      "longitude": 11.723147
+    },
+    "url_id": 423
+  },
+  {
+    "name": "STUBISTRO_ARCISSTR",
+    "long_name": "StuBistro Arcisstra\u00dfe",
+    "site": {
+      "address": "Leopoldstra\u00dfe 13A, M\u00fcnchen",
+      "latitude": 48.156486,
+      "longitude": 11.581872
+    },
+    "url_id": 450
+  },
+  {
+    "name": "STUBISTRO_GOETHESTR",
+    "long_name": "StuBistro Goethestra\u00dfe",
+    "site": {
+      "address": "Goethestra\u00dfe 70, M\u00fcnchen",
+      "latitude": 48.131396,
+      "longitude": 11.558264
+    },
+    "url_id": 418
+  },
+  {
+    "name": "STUBISTRO_GROSSHADERN",
```

Comment:
I mentioned the issue with the `ß` in this canteen-id in issue #68.
When we change it here, we also need to move the files in the gh-pages as I described in the issue